src/views/CourtOrder.vue

```diff
@@ -0,0 +1,696 @@
+                @total-fee="totalFee=$event"
+              />
+              <v-card-actions class="w-full max-w-3xl mx-auto">
+                <div class="form__btns w-full">
+                  <v-col>
+                    <v-btn
+                      id="dialog-cancel-button"
+                      outlined
+                      color="primary"
+                      class="btn-outlined-primary mt-6 text-lg px-6 py-3"
+                      :disabled="saving"
+
+                      @click="goToDashboard()"
+                    >
+                      <span>Cancel</span>
+                    </v-btn>
+                    <div class="buttons-right">
+                      <v-btn
+                        id="dialog-save-button"
+                        solid
+                        color="primary"
+                        class="btn-outlined-primary mt-4"
+                        :loading="!isPageValid || saving"
+                        @click.native="onSave()"
+                      >
+                        {{ isPayRequired ? "File and Pay" : "File Now (no fee)" }}
+                      </v-btn>
+                    </div>
+                  </v-col>
+                </div>
+              </v-card-actions>
+            </affix>
+          </aside>
+        </v-col>
+      </v-row>
+    </v-container>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Mixins, Prop, Vue, Watch } from 'vue-property-decorator'
+import { Getter } from 'pinia-class'
+import { DateMixin, FilingMixin, CommonMixin } from '@/mixins'
+import { CourtOrderPoa } from '@bcrs-shared-components/court-order-poa'
+import FileUploadPdf from '@/components/common/FileUploadPdf.vue'
+import { FormIF, StaffPaymentIF } from '@/interfaces'
+import { EffectOfOrderTypes, PageSizes } from '@/enums'
+import { FilingCodes, FilingNames, FilingTypes, StaffPaymentOptions } from '@bcrs-shared-components/enums'
+import { EnumUtilities, LegalServices } from '@/services'
+import { useAuthenticationStore, useBusinessStore, useConfigurationStore, useRootStore } from '@/stores'
+import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
+import SbcFeeSummary from 'sbc-common-components/src/components/SbcFeeSummary.vue'
+import { StaffPayment as StaffPaymentShared } from '@bcrs-shared-components/staff-payment/'
+
+@Component({
+  components: {
+    CourtOrderPoa,
+    FileUploadPdf,
+    SbcFeeSummary,
+    StaffPaymentShared
+  }
+})
+export default class CourtOrderView extends Mixins(DateMixin, FilingMixin, CommonMixin) {
+  $refs!: Vue['$refs'] & {
+    courtOrderPoaRef: FormIF,
+    fileUploadRef: FormIF,
+    notationFormRef: FormIF
+  }
+
+  readonly MAX_FILE_SIZE = 30 // in MB
+  readonly NOTATION_MAX_LENGTH = 2000 // characters
+
+  // For template
+  readonly LegalServices = LegalServices
+  readonly PageSizes = PageSizes
+  // enum for template
+  readonly FilingCodes = FilingCodes
+  /** Prop to display the dialog. */
```

Comment:
Are any of these props needed? If not, please replace remove them.

Reply:
You didn't fix this here so it will have to be updated later. (Ideally, when you write the unit tests.)